KsApi/ServiceTypeTests.swift

```diff
@@ -319,15 +319,11 @@ final class ServiceTypeTests: XCTestCase {
   }
 }
 
-// swiftformat:disable wrap
 private func userAgent() -> String {
-  return """
-  xctest/\(testToolBuildNumber()) (\(UIDevice.current.model); iOS \(UIDevice.current.systemVersion) Scale/\(UIScreen.main.scale))
-  """
+  // swiftformat:disable:next wrap
```

Comment:
This one _does_ work, but I was unable to apply it to other files ❓😕❓ Something strange is going on here and I don't have the bandwidth to dive into it.